superset-frontend/plugins/plugin-chart-echarts/src/constants.ts

```diff
@@ -71,18 +71,29 @@ export enum OpacityEnum {
   NonTransparent = 1,
 }
 
-export enum AreaChartExtraControlsValue {
+export enum StackControlsValue {
   Stack = 'Stack',
+  Stream = 'Stream',
   Expand = 'Expand',
 }
 
+export const ExtraControlsOptions: [
+  JsonValue,
+  Exclude<ReactNode, null | undefined | boolean>,
+][] = [
+  [null, t('None')],
+  [StackControlsValue.Stack, t('Stack')],
+  [StackControlsValue.Stream, t('Stream')],
+];
+
 export const AreaChartExtraControlsOptions: [
   JsonValue,
   Exclude<ReactNode, null | undefined | boolean>,
 ][] = [
   [null, t('None')],
-  [AreaChartExtraControlsValue.Stack, t('Stack')],
-  [AreaChartExtraControlsValue.Expand, t('Expand')],
+  [StackControlsValue.Stack, t('Stack')],
+  [StackControlsValue.Stream, t('Stream')],
+  [StackControlsValue.Expand, t('Expand')],
 ];
```

Comment:
Maybe a nit, but this had me confused: could these be called `StackControlOptions` and `AreaChartStackControlOptions`?

Reply:
While we're at it, could we spread the former into the latter to DRY it up?